force-app/main/default/profiles/Admin.profile-meta.xml

```diff
@@ -84,6 +89,10 @@
         <object>Template__c</object>
         <viewAllRecords>true</viewAllRecords>
     </objectPermissions>
+    <tabVisibilities>
```

Comment:
If you can delete this reference, it will cause problems with your deployment in the future.

Reply:
Deleted.